### ISSUE TITLE
AWESOME Numismatic Suggestions ✨💹

### ISSUE THREAD

**Issue by Mostovm** [glisco03/numismatic-overhaul#141]
More of GOOD suggestions that I think will make mod stunning and more accepting;

-Base factor for how much the money should be valued and exchanged between villagers instead of a fixed value. (CONFIG)
-Option to keep money regardless of keep inventory. I want to keep money definite for my server. (CONFIG)
-Config for allowing or disabling items and blocks such as Piggy Bank as I don't see this very useful in my case as I want the option to have their currency always saved on the player instead of a block, it just gets annoying just to keep track of blocks that have a specific 
          function when it can just be in the players inventory. (CONFIGURATIONS)
-Please revamp textures I would appreciate them more shinier. So as shop blocks they could use a makeover. Piggy bank block looks 
          immaculate though 👌. (TEXTURE)
-Fix withdrawal when grabbing money from your purse, I'd rather just click it out then use the little buttons to take money. Very tedious 
          and annoying (FUNCTION)
-Auction block or your shop block should be a little more organized and seems glitchy. It should also be labled like a chest would at the top, I think this is godly for multiplayer. Please allow overall function of selling items into a catalog for others to view, & buy with items 
           or currency if possible. (FUNCTION)
-Picking up coins go directly into purse not inventory. When taking out coins they go in your inventory until thrown. (FUNCTION)

Modding overall is about options and allowing modifications to peoples want's but I would deeply appreciate your consideration on these suggestions and implement what I think will make this an AWESOME money mod.
🙏

**Comment by Noaaan** [comment]
1. Having a base value is currently not something we want to implement, as villager trades are static. Once generated their trades (including their prices) do not really change. 
The current design of trades match up with vanilla: regular trades have static prices, while special trades, like enchanted books, vary depending on the enchantment. 
Your best bet to keep a stable economy is to keep sources of cash in check - disable/ban raid farms (as by default these drop coins and can easily be grinded up. You can change this with the respective tag as mentioned on [the wiki](https://docs.wispforest.io/numismatic-overhaul/home/#mob-drops)), and introduce some coin sinks (maybe by using inexhaustible shops).
3. This is already a feature. As documented on the wiki: setting the `moneyDropPercentage` gamerule will let you configure how much money is dropped on death
4. There are many tools that can effectively disable the recipe for Piggy Banks, which would make them unobtainable. CraftTweaker, KubeJS, even regular datapacks. I can agree the piggy bank isn't exactly versatile, but you always have the option to not craft it.
5. We currently have no plans to rework any textures
6. I do not quite understand this question. I will mention that the purse has a lot of handy shortcuts, as written here: https://docs.wispforest.io/numismatic-overhaul/home/#useful-shortcuts
7. Auction Houses kinda goes against the spirit of Numismatic Overhaul - which is adding items used for bartering between players. This is not something we are considering adding. It isn't meant to be a drop-in replacement for an Economy plugin. You might be able to set this up yourself using some mods, like [AuctionHouse](https://modrinth.com/mod/auctionhouse#supported-economies), [EightsEconomy](https://legacy.curseforge.com/minecraft/mc-mods/eightseconomyp), and [CommonBridge](https://modrinth.com/mod/common-bridge)
8. We have been discussing on adding this feature, and are currently considering it

**Comment by notIkibo** [comment]
Number 7 slaps hard. Generally this is one of the most important parts. Because mods need to strive to be as convenient as possible. Sure you can shift click your purse for instant purse collection, but when killing some pillagers, you want it to be in your purse. Like ALWAYS. If you find moneybags in chests, you actually also always want it in your purse. This is harder to do, but this whole inventory thing can feel disruptive. There has to be an elegant solution. Though this also means somewhat specific behaviour coding that doesn't exist in mc by default.

At least number 7 should be heavily considered and implemented.

**Comment by A5ho9999** [comment]
I agree with the number 2, I do think it should be a config option over gamerule. Especially when a lot of them require cheats to be enabled in the first place. Having it has a quick config option would make it easier for modpack developers as well. It's for sure something that should be considered

**Comment by FlameFire26** [comment]
> Number 7 slaps hard. Generally this is one of the most important parts. Because mods need to strive to be as convenient as possible. Sure you can shift click your purse for instant purse collection, but when killing some pillagers, you want it to be in your purse. Like ALWAYS. If you find moneybags in chests, you actually also always want it in your purse. This is harder to do, but this whole inventory thing can feel disruptive. There has to be an elegant solution. Though this also means somewhat specific behaviour coding that doesn't exist in mc by default.
> 
> At least number 7 should be heavily considered and implemented.
> 
> 

Exactly.

Maybe the author think that it is a way to separate your purse and coins get newly (some players want to know how much they get)

So it could be nice to add a config to **let picked coins auto combine into money bag** (rather than bothering to click to combine one by one)

The two points can make a chosen config, so you can set directly go to purse or keep in inventory with combined money bag.

A little wish: I'm working on a modpack in 1.20.1, could it be added to 1.20.1 version?
